### ISSUE TITLE
strange value ....

### ISSUE THREAD

**Issue by Eddie-BS** [jakenl/domoticz_solarbattery#6]
This is a really strange value ... should be somewhere around ZERO ...

<img width="387" alt="Screenshot 2021-05-03 at 21 57 12" src="https://user-images.githubusercontent.com/10174704/116926428-a201e780-ac5a-11eb-907b-725558d20edb.png">

**Comment by jakenl** [comment]
Wow, that is a crazy value indeed! Have you been able to pinpoint it to a specific time of the day that it happened, or did it have big chunks of lost energy over the day?
Although a little premature, I assume a dividing calculation has gone wrong at some point?

**Comment by Eddie-BS** [comment]
Not really, indeed some dividing might be suspicious, but which one ... 
It caught my attention just yesterday ... 
<img width="1173" alt="Screenshot 2021-05-04 at 07 34 13" src="https://user-images.githubusercontent.com/10174704/116964244-5592c800-acab-11eb-89c4-2eb81c7d4f2e.png">
